policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -488,6 +488,15 @@ fn parse_accrual_config(
                         "min_penalty ({min_penalty:?}) cannot exceed max_penalty ({max_penalty:?})",
                     );
                 }
+                if max_penalty == time::Duration::from_millis(0) {
+                    bail!("max_penalty cannot be zero");
+                }
+                if jitter < 0.0 {
+                    bail!("jitter cannot be negative");
+                }
+                if jitter > 100.0 {
+                    bail!("jitter cannot be greater than 100");
+                }
 
```

Comment:
let's also add a check validating that the `jitter` value is not infinity, similarly to [the one in the proxy](https://github.com/linkerd/linkerd2-proxy/blob/846ce1b5243fcba8958414ae732156f3b1639747/linkerd/exp-backoff/src/lib.rs#L68-L70).

i wasn't totally sure if this was necessary or not, since we are parsing floats from a string, and I wasn't sure `f64::parse` will parse strings like "inf", but i went ahead and checked, and it looks like it will: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d70ed1e3f65505cf71752257c9144968
so, we should reject infinities here, probably.

Reply:
I think the `jitter > 100.0` check already covers that jitter is not infinity, no?
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=aab532d119f4ea103f2eb8ac67d73634

Reply:
...yup, i didn't think this through :)

the check that it's greater than 0 should also handle the case where the jitter value is `-inf`, so this is fine as-is.